cmd/remove.go

```diff
@@ -30,15 +29,20 @@ adjust this cap using IPFS, qri will respect it.
 
 In the future we’ll add a flag that’ll force immediate removal of a dataset from
 both qri & IPFS. Promise.`,
-		Example: `  remove a dataset named annual_pop:
+		Example: `  # Remove a dataset named ` + "`annual_pop`" + `:
   $ qri remove me/annual_pop --all`,
 		Annotations: map[string]string{
 			"group": "dataset",
 		},
+		// Use *max* so we can print a nicer message for no or malformed args
+		Args: cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if err := o.Complete(f, args); err != nil {
 				return err
 			}
+			if err := o.Validate(); err != nil {
+				return err
+			}
```

Comment:
`Validate()` previously wasn’t getting called at all (though it was tested), so I added it.